srtcore/core.cpp

```diff
@@ -3302,6 +3297,9 @@ void CUDT::startConnect(const sockaddr *serv_addr, int32_t forced_isn)
 
     CUDTException  e;
     EConnectStatus cst = CONN_CONTINUE;
+    // This is a temporary place to store the DESTINATION IP from the incoming packet.
+    // We can't record this address yet until the cookie-confirmation is done, for safety reasons.
+    sockaddr_any use_source_adr(m_iIPversion); // use BindAddress.family() after refactoring
```

Comment:
> // use BindAddress.family() after refactoring

Can `BindAddress.family()` be already used?

Reply:
No. The refactoring is only available in `dev-groups-unified-sockaddr`.